config/application.yml.default

```diff
@@ -325,6 +325,7 @@ show_user_attribute_deprecation_warnings: false
 otp_min_attempts_remaining_warning_count: 3
 system_demand_report_email: 'foo@bar.com'
 sp_issuer_user_counts_report_configs: '[]'
+team_ursula_email: ''
```

Comment:
I feel like as long as the team exists, it makes sense to call it that. If say, we switch this responsibility to team XYX in the future, we'd add an alias for that? I think that the pattern of notifying a specific team for a specific thing in the app could continue into the future. And I'd rather have each team's notification email defined once, rather than like duplicated across say `extra_sp_notification_team` and `partner_api_notification_team`